<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Equipements</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Petition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset ; 
border-width:0px ; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_petition (1).png</normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_petition (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Materiel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset ; 
border-width:0px ; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_materiel.png</normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_materiel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Crimes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset ; 
border-width:0px ; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_crimes.png</normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_crimes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Agent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset ; 
border-width:0px ; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_agents.png</normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_agents.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plaintes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>377</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset ; 
border-width:0px ; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_plaintes.png</normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_plaintes.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Mission">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset ; 
border-width:0px ; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_missions.png</normaloff>C:/Users/zraib/OneDrive/Bureau/ifinal/button_missions.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>921</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>800</width>
        <height>481</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Equipement</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>781</width>
          <height>401</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string/>
         </attribute>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>769</width>
            <height>409</height>
           </rect>
          </property>
          <property name="title">
           <string>ManipulerEquipement</string>
          </property>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>id</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>70</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>nom</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>40</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>type</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>70</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>quantit√©</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>40</y>
             <width>55</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>etat</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_20">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>70</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>dateFabrication</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_id">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>40</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_nom">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>70</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_type">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>40</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="la_quantite">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>70</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_ajouter">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>110</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
          <widget class="QTableView" name="tab_equipement">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>230</y>
             <width>681</width>
             <height>151</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_supprimer">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>110</y>
             <width>91</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_modifier">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>110</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <property name="text">
            <string>modifier</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_equipement">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>200</y>
             <width>73</width>
             <height>22</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <item>
            <property name="text">
             <string>etatEquipement</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>quantiteEquipement</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="trier">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>51</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <property name="text">
            <string>Trier</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="rech_equipement">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>170</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_rechercher">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>200</y>
             <width>81</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <property name="text">
            <string>rechercher</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_pdf">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>200</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <property name="text">
            <string>Generate_pdf</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_22">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>270</y>
             <width>141</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_statistique">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>200</y>
             <width>81</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px; </string>
           </property>
           <property name="text">
            <string>Statistiques</string>
           </property>
          </widget>
          <widget class="QPushButton" name="imprimer">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>200</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
           </property>
           <property name="text">
            <string>imprimer</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_icon_upload">
           <property name="geometry">
            <rect>
             <x>690</x>
             <y>250</y>
             <width>51</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-style:outset ; 
border-width:0px ; 
border-radius:0px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>C:/Users/zraib/Downloads/upload_icon.png</normaloff>C:/Users/zraib/Downloads/upload_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
          </widget>
          <widget class="QLineEdit" name="le_etat">
           <property name="geometry">
            <rect>
             <x>540</x>
             <y>40</y>
             <width>113</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QDateEdit" name="la_date">
           <property name="geometry">
            <rect>
             <x>540</x>
             <y>70</y>
             <width>110</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Statistique</string>
         </attribute>
         <widget class="QFrame" name="frame_charts">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>0</y>
            <width>791</width>
            <height>431</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Maintenance</string>
       </attribute>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>identifiant</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>dureeM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>dateEntree</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>dateSortie</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>frais</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_maintenance">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="duree_maintenance">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="date_entree_maintenance">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="date_sortie_maintenance">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="frais_maintenance">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tab_maintenance">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>300</y>
          <width>691</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer_maintenance">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>260</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouterMaintenance">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_sup">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>210</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>210</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_maintenance">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>240</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white; 
color: rgb(255, 0, 255);
border-style:outset ; 
border-width:3px ; 
border-radius:10px;</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>150</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>fk_equipement</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="fk_equipement_equipement">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
    <widget class="QWidget" name="page_6"/>
    <widget class="QWidget" name="page_7"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/zraib/OneDrive/Bureau/ifinal/Logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionprint"/>
    <addaction name="actionQuit"/>
    <addaction name="actionGenerate_pdf"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionprint"/>
   <addaction name="actionQuit"/>
   <addaction name="actionGenerate_pdf"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionprint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/print_icon.png</normaloff>:/img/print_icon.png</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/quit_icon.png</normaloff>:/img/quit_icon.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionGenerate_pdf">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/pdf_icon.png</normaloff>:/img/pdf_icon.png</iconset>
   </property>
   <property name="text">
    <string>Generate_pdf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
